ServiceApp/src/org/droidplanner/services/android/core/gcs/follow/LocationRelay.java

```diff
@@ -0,0 +1,121 @@
+package org.droidplanner.services.android.core.gcs.follow;
+
+import android.content.Context;
+
+import com.o3dr.services.android.lib.coordinate.LatLongAlt;
+
+import org.droidplanner.services.android.core.gcs.location.Location;
+
+import timber.log.Timber;
+
+/**
+ * Created by kellys on 2/24/16.
+ */
+public class LocationRelay {
```

Comment:
Ah, good call. I had forgotten I had replicated much of FusedLocation in
LocationRelay. Updated.


On Thu, Jun 23, 2016 at 11:01 AM, Fredia Huya-Kouadio <
notifications@github.com> wrote:

> In
> ServiceApp/src/org/droidplanner/services/android/core/gcs/follow/LocationRelay.java
> <https://github.com/dronekit/dronekit-android/pull/410#discussion_r68262891>
> :
>
> > @@ -0,0 +1,121 @@
> > +package org.droidplanner.services.android.core.gcs.follow;
> > +
> > +import android.content.Context;
> > +
> > +import com.o3dr.services.android.lib.coordinate.LatLongAlt;
> > +
> > +import org.droidplanner.services.android.core.gcs.location.Location;
> > +
> > +import timber.log.Timber;
> > +
> > +/**
> > + * Created by kellys on 2/24/16.
> > + */
> > +public class LocationRelay {
>
> @kellyschrock <https://github.com/kellyschrock> this class seems to
> replicate a lot of the functionality in the FusedLocation. I think it
> makes sense for the functionality to be here, so we should remove it from
> the FusedLocation class, and have the FusedLocation class call this class
> instead and pass the updated locations through here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dronekit/dronekit-android/pull/410/files/8151ddc9258dbe0add93692860b8cf03b32e8284#r68262891>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADmhzeSXruze1aOszRS5GyhPx2OMllJjks5qOq3jgaJpZM4I7YHE>
> .
>